### ISSUE TITLE
Finetune features

### ISSUE THREAD

**Issue by hitcbw** [ttlmh/Bridge-Prompt#6]
Hi,
Thanks for your amazing work. Here is a question: how to fine-tune the feature extractor （ViT/b）on the specific dataset (like GTEA)? I suppose that the video with the segmentation label will be split into various segments and each segment should be treated as a trimmed action video. I want to know whether I am right :)
By the way, would you please release fine-tuned model? Thanks for your efforts.

**Comment by ttlmh** [comment]
Hi, thanks for your interest in our work.

As for the fine-tuning procedure, please refer to the **Training** section in the README file. Each raw instructional video is split into multiple equal-length segments (windows) for training, and you are right about that.

At test time, we have trained Br-Prompt models on all splits for each dataset. To facilitate your testing, we upload the specific Br-Prompt model trained on split-1 of the GTEA dataset, which you can download from [here](https://pan.baidu.com/s/1uWf867SwEMwlhFeUGfpmJw?pwd=3fzh). You should change the relevant parameters for testing.

Thanks!

**Comment by ttlmh** [comment]
We have already uploaded the trained weights! Please check the updated README file.

**Comment by hitcbw** [comment]
Thanks for your kind reply. Your work is excellent.
